File: Modul 3/Bahasa Python/PRAK305-2310817120001-DinaIzzatiElfadheya.py
print("Input")
total_detik = int(input())

hari = total_detik // (24 * 3600)
jam = (total_detik % (24 * 3600)) // 3600
total_detik %= 3600
menit = total_detik // 60
detik = total_detik % 60

print("\nOutput")

if hari > 0:
    print(f"{hari} hari {jam:02d}:{menit:02d}:{detik:02d}")
else:
    print(f"{jam:02d}:{menit:02d}:{detik:02d}")

File: Modul 3/Bahasa C/PRAK305-2310817120001-DinaIzzatiElfadheya.c
#include <stdio.h>
int main() {

    int total_detik, hari, jam, menit, detik;
    printf("Input\n");
    scanf("%d", &total_detik);

  hari = total_detik/ (24*3600);
  jam = total_detik % (24*3600)/3600;
  total_detik = total_detik % 3600; 
  menit = total_detik / 60;
  detik = total_detik % 60;

    printf("\nOutput\n");
  if (hari > 0){
    printf("%d hari %02d:%02d:%02d", hari, jam, menit, detik);
  } else {
    printf("%02d:%02d:%02d", jam, menit, detik);
  }
  return 0;
}